api/segments/models.py

```diff
@@ -70,6 +80,15 @@ def does_identity_match(
             rule.does_identity_match(identity, traits) for rule in rules
         )
 
+    def get_create_log_message(self, history_instance) -> typing.Optional[str]:
```

Comment:
Well, technically it should I think, because the method it should keep the signature of the method it's overriding. So, even though this implementation of the method can never return None, I think that, because the parent method defines it as such, it should maintain that. Perhaps I'm wrong here though?

Reply:
Hmm, Yeah, good point. But I think it only makes the code more confusing(I'd have to and check the parent class, unless it is a convention?) without adding anything? I am fine either way though